server/vulnerabilities/oval/sync.go

```diff
@@ -103,19 +109,19 @@ func Sync(dstDir string, platforms []Platform) error {
 	for _, platform := range platforms {
 		defFile, err := downloadDefinitions(sources, platform, dwn)
 		if err != nil {
-			return err
+			return fmt.Errorf("downloadDefinitions: %w", err)
 		}
 
 		dstFile := strings.Replace(filepath.Base(defFile), ".xml", ".json", 1)
 		dstPath := filepath.Join(dstDir, dstFile)
 		err = parseDefinitions(platform, defFile, dstPath)
 		if err != nil {
-			return err
+			return fmt.Errorf("parseDefinitions: %w", err)
 		}
 
 		err = os.Remove(defFile)
 		if err != nil {
-			return err
+			return fmt.Errorf("remove: %w", err)
```

Comment:
nit: should we print what are we trying to remove? maybe it's part of the go error?

Reply:
good catch!